.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     - uses: actions-rs/toolchain@v1
       with:
           profile: minimal
-          toolchain: nightly
+          toolchain: stable
           override: true
           components: rustfmt
```

Comment:
Changed this because CI failed on a formatting difference between stable and nightly. Since this crate is for stable rust it makes more sense to me to use stable formatting.